### PR TITLE
fix:  styles in signup page , footer and headings

### PR DESCRIPTION
**Pull request by Srish-ty**

### CODE REVIEW

**Review by Shurtu-gal — COMMENTED** [review]
@Srish-ty Input boxes size can be increased.

**Comment by Srish-ty** [comment]
> @Srish-ty Input boxes size can be increased.

It was showing fine, but after merging new commits from origin, it's height has changed

**Comment by Srish-ty** [comment]
@Shurtu-gal , @ashutosh-rath02 please review

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]
@Srish-ty Build is failing

**Comment by Srish-ty** [comment]
> @Srish-ty Build is failing

I'm able to build locally

**Comment by Srish-ty** [comment]
@Shurtu-gal , @ashutosh-rath02 please review

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]
image
Some solution such as ellipsis or sth else.

image
- The close icon is still not good, it can be made smaller and put on corner of modal.
- Clicking on backdrop should close the modal
- The team name is getting cut.

**Comment by Srish-ty** [comment]
> image Some solution such as ellipsis or sth else.
> 
> image
> 
> * The close icon is still not good, it can be made smaller and put on corner of modal.
> * Clicking on backdrop should close the modal
> * The team name is getting cut.

Fixed that styling. [commited changes](https://github.com/dscnitrourkela/project-ticket/pull/14/commits/3d58f2118a53fdeb88a923eb126559f142dda637)

**Comment by Srish-ty** [comment]
fixed preview for sharing on socials. 

[commit link](https://github.com/dscnitrourkela/project-ticket/pull/14/commits/e956e157f065f9dc5aca2381428f0d3694928a3a)
image

File: src/app/preview/[slug]/opengraph-image.tsx [hunk 1]
```diff
@@ -1,106 +1,38 @@
...
-        />
-        <div style={{ ...ticketTextStyle } as React.CSSProperties}>
-          <div style={{ ...namesStyle } as React.CSSProperties}>
-            <h1 style={userNameStyle}>{name || ''}</h1>
-            <h2 style={{ ...teamNameStyle } as React.CSSProperties}>{teamName || ''}</h2>
-          </div>
-          <span style={{ ...ticketNumStyle } as React.CSSProperties}>{ticketId || '550000'}</span>
-        </div>
-      </div>
+      <InnerTicket
```
Comment (Shurtu-gal) [inline src/app/preview/[slug]/opengraph-image.tsx]: Is this working ?
Because I had tried this and it wasn't working.

Comment (Srish-ty) [reply src/app/preview/[slug]/opengraph-image.tsx]: Yeah, I've changed the code.
And now the preview is visible, I'd attached the screenshot.

Comment (Shurtu-gal) [reply src/app/preview/[slug]/opengraph-image.tsx]: This is not for that. The link for this goes like http://localhost:3000/preview/B0uHy0AIhcQWVv768GLl8auhXBm2/opengraph-image. And is used for of image generation.

@vercel/og does not support the components written before.

Comment (Srish-ty) [reply src/app/preview/[slug]/opengraph-image.tsx]: fixed. [commit](https://github.com/dscnitrourkela/project-ticket/pull/14/commits/dbc2ad5f1e10e4a9a1620fddbdfcd3dfddb42f17)

**Review by Shurtu-gal — APPROVED** [review]
LGTM 🪨

**Pull request merged**